facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java

```diff
@@ -46,14 +47,8 @@
     private BuildConfigurationRepository buildConfigurationRepository;
 
     @Inject
-    public BuildProviderImpl(BuildRecordRepository repository,
-                             BuildMapper mapper,
-                             EntityManager entityManager,
-                             BuildConfigurationRepository buildConfigurationRepository) {
-
+    public BuildProviderImpl(BuildRecordRepository repository, BuildMapper mapper) {
```

Comment:
The entityManager and buildConfigurationRepository are nto needed anymore? If not delet the fields too.

Reply:
BuildConfigurationRepository still needed. Sorry for that error!